client/scss/utilities/variables_config/breakpoints.js

```diff
@@ -0,0 +1,8 @@
+const containerWidthMax = require('./container-width-max');
+
+module.exports = {
+  small: '0',
+  medium: '600px',
+  large: '960px',
+  xlarge: containerWidthMax
+};
```

Comment:
I'm going to be pedantic - but realised this is in `scss/config` - where it is actually common config as the JS uses it too - could we potentially just store it in `config`?